src/frontend/apps/desk/src/api/APIError.ts

```diff
@@ -0,0 +1,17 @@
+interface IAPIError {
+  status: number;
+  cause?: string[];
```

Comment:
Should we keep it plural ? As it's an array with multiple causes.

Reply:
I was going in this way, but APIError extends `Error`, and `Error` has in his interface already a prop named `cause`, used for what we want, so instead of creating another prop (causes), I reuse the props provided by the Error class.
Here an example to illustrate my point:
![Capture d'écran 2024-02-22 154032](https://github.com/numerique-gouv/people/assets/25994652/97fe8212-a872-4b6e-9957-5f6ff4e0f8c8)
![Capture d'écran 2024-02-22 154044](https://github.com/numerique-gouv/people/assets/25994652/00d446ae-f495-49ee-834b-74e11e70849f)